docs/sources/introduction/backward-compatibility.md

```diff
@@ -8,7 +8,7 @@ weight: 999
 
 # {{% param "FULL_PRODUCT_NAME" %}} backward compatibility
 
-{{< param "FULL_PRODUCT_NAME" >}} follows [semantic versioning][].
+{{< param "FULL_PRODUCT_NAME" >}} mostly follows [semantic versioning][], with the differences documented in the [Exceptions](#exceptions) section.
```

Comment:
```suggestion
{{< param "FULL_PRODUCT_NAME" >}} follows [semantic versioning][]. [Exceptions](#exceptions) may arise that require a breaking change without a new major version.
```

I think we should avoid the vague "mostly" and just state we use semver, but exceptions can happen.